cmd/utils.go

```diff
@@ -350,7 +372,52 @@ func getProjectName(config *RootCommandConfig) (string, error) {
 	v := viper.New()
 	v.SetConfigFile(appsodyConfig)
 	err = v.ReadInConfig()
-	return v.GetString("project-name"), err
+
+	if err != nil {
+		return "my-project", err
+	}
+
+	projectName := v.GetString("project-name")
+
+	if projectName != "" {
+		match, err := regexp.MatchString("[a-z]([-a-z0-9]*[a-z0-9])?", projectName)
+
+		if !match {
+			v.Set("project-name", "my-project")
```

Comment:
I thought we agreed that if this isn't a match we would just error out with a message that explains that the project name must conform to those rules. Here we are changing it to `my-project` if I am not mistaken.

Reply:
changed